browser/brave_ads/android/java/org/chromium/chrome/browser/dialogs/BraveAdsSignupDialog.java

```diff
@@ -138,9 +140,9 @@ public static void showExistingUserDialog(Context context) {
             public void onClick(DialogInterface dialog, int which) {
                 // Enable ads
                 neverShowOnboardingDialogAgain();
-
-                BraveRewardsNativeWorker.getInstance().CreateRewardsWallet();
-                BraveAdsNativeHelper.nativeSetAdsEnabled(Profile.getLastUsedRegularProfile());
+                if (BraveActivity.getBraveActivity() != null) {
+                    BraveActivity.getBraveActivity().openRewardsPanel();
```

Comment:
Looks like BraveActivity.getBraveActivity() costly operation
Here called twice
Better collect once in a variable